src/commands/helpcenter.js

```diff
@@ -0,0 +1,47 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { MessageEmbed } = require('discord.js');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('helpcenter')
+    .setDescription('Provies information to access the helcenter!')
```

Comment:
Minor typos here: `Provies` -> `Provides` and `helcenter` to `helpcenter`.

Reply:
Thanks! I definately need a VSCode spellchecker 😅

Reply:
@lyallstewart I think this is the one I use
https://marketplace.visualstudio.com/items?itemName=ban.spellright

Reply:
Awesome, thanks!